src/style.css

```
@@ -0,0 +1,24 @@
+@import "./card.css";
+@import "./stars.css";
+
+:root {
+  --color-main-accent: #060b35;
+  --color-secondary: #616070;
+  --color-blue-accent: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.body,
```

Can use tag for this.
```suggestion
body,
```